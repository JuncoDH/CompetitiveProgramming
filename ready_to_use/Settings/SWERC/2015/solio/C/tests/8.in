20
1
1
2
1 1
3
2 1 1
4
3 1 1 2
5
1 3 1 2 5
6
1 8 2 3 5 1
7
3 2 13 1 8 1 5
8
2 3 21 1 13 1 8 5
9
1 8 2 21 5 13 1 34 3
10
2 13 55 1 5 1 34 3 21 8
11
34 3 1 55 13 2 89 8 21 5 1
12
13 2 8 1 1 89 3 21 55 144 34 5
13
89 34 21 55 233 1 13 3 8 144 5 2 1
14
1 377 55 3 144 21 2 5 13 233 8 34 1 89
15
1 8 233 5 610 2 13 144 34 3 377 55 89 1 21
16
55 1 5 610 21 377 233 13 1 2 3 89 144 987 34 8
17
8 233 2 13 21 34 55 89 5 3 144 1597 1 987 610 1 377
18
610 2 2584 55 3 987 1 5 8 1 34 21 1597 89 377 13 233 144
19
5 610 2 3 4181 55 233 987 13 1597 21 89 377 1 8 144 1 2584 34
20
1597 2584 21 610 987 144 377 13 55 6765 34 8 233 4181 3 5 1 89 2 1
