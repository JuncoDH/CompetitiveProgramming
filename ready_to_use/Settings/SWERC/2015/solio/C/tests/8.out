0
2
6
13
25
45
78
132
220
363
595
971
1580
2566
4162
6745
10925
17689
28634
46344
